app/presenters/hyrax/file_set_presenter.rb

```diff
@@ -25,7 +25,7 @@ def initialize(solr_document, current_ability, request = nil)
 
     # Metadata Methods
     delegate :title, :label, :description, :creator, :contributor, :subject,
-             :publisher, :language, :date_uploaded, :rights,
+             :publisher, :language, :date_uploaded,
```

Comment:
It was delegated, but then the delegate was overridden below, so it was never necessary to delegate it.